app/src/main/java/study_dev/testbed/studyhopper/models/Member.java

```diff
@@ -0,0 +1,73 @@
+package study_dev.testbed.studyhopper.models;
+
+import com.google.firebase.Timestamp;
+
+public class Member {
+
+    private String firstName;
+    private String lastName;
+    private String userDocId;
+    private String screenName;
+    private boolean isOwner;
+    private Timestamp whenJoined;
```

Comment:
Maybe we can also store the ref to where the corresponding document is stored in the user->group collection to make it easier to keep track.

Reply:
Added a  userGroupDocId variable to store this value.